<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShipWindow</class>
 <widget class="QDialog" name="ShipWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отгрузка</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid #9F9F9F;
    font-size: 14px;
    font-weight: 800;
    color: black;
    border-radius: 4px;
    margin-top: 2.5ex; 
    padding-top: 2ex;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
}


QPushButton {
    border-radius: 3px;
    border: 1px solid #9F9F9F;
	padding-left: 8px;
	padding-right: 8px;
	padding-top: 4px;
	padding-bottom: 4px;
}

QPushButton:hover { 
	background-color: white;
}

QPushButton:pressed {
background-color: LightGray;
    position: relative;
     top: 4px; left: 4px;
}


</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Состав отгрузки</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QLineEdit" name="leNumModul"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Модуль</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="tbAddSetterProd">
          <property name="toolTip">
           <string>Добавить изделие  нвборе</string>
          </property>
          <property name="text">
           <string>в набор</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/image/Add2.png</normaloff>:/image/Add2.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="leNumProd"/>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="tbNumProd">
          <property name="toolTip">
           <string>Добавить изделие</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/image/Add2.png</normaloff>:/image/Add2.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Изделие</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="tbNumModul">
          <property name="toolTip">
           <string>Добавить модуль</string>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/image/Add2.png</normaloff>:/image/Add2.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="leSetterOut"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Набор</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="tbAddSetter">
          <property name="text">
           <string>+</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/image/Add2.png</normaloff>:/image/Add2.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QToolButton" name="tbAddSetProd">
          <property name="text">
           <string>с изделием</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/image/Add2.png</normaloff>:/image/Add2.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbDelete">
          <property name="toolTip">
           <string>Удалить выделенную строку</string>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/image/cancel2.png</normaloff>:/image/cancel2.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="twTreeDevice">
        <column>
         <property name="text">
          <string>Наименование</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Объект установки</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leObjectInstall"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Дата отгрузки</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="deDateOut">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Заказчик</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cbCusomer"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Счет</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="leSchet"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Карта заказа</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="leCardOrder"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Покупатель</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="leBuyer"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>№ УПД</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="leNumUPD"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Дата УПД</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QDateEdit" name="deDateUPD">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignBottom">
        <widget class="QPushButton" name="pbFinish">
         <property name="text">
          <string>Отгрузить</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/image/lorry.png</normaloff>:/image/lorry.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignmentFlag::AlignBottom">
        <widget class="QPushButton" name="pbClose">
         <property name="text">
          <string>Закрыть</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/image/Exit.png</normaloff>:/image/Exit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>twTreeDevice</tabstop>
  <tabstop>leObjectInstall</tabstop>
  <tabstop>deDateOut</tabstop>
  <tabstop>leSchet</tabstop>
  <tabstop>leCardOrder</tabstop>
  <tabstop>leBuyer</tabstop>
  <tabstop>leNumUPD</tabstop>
  <tabstop>deDateUPD</tabstop>
  <tabstop>leNumProd</tabstop>
  <tabstop>leNumModul</tabstop>
  <tabstop>pbFinish</tabstop>
  <tabstop>pbClose</tabstop>
  <tabstop>tbNumModul</tabstop>
  <tabstop>pbDelete</tabstop>
  <tabstop>tbNumProd</tabstop>
  <tabstop>tbAddSetterProd</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
